engine/src/main/scala/cromwell/engine/workflow/WorkflowDockerLookupActor.scala

```diff
@@ -289,4 +314,10 @@ object WorkflowDockerLookupActor {
       */
     def withFailureCause(cause: Throwable): WorkflowDockerLookupActorData = this.copy(failureCause = Option(cause))
   }
+
+  implicit class EnhancedNonEmptyList[A](val nel: NonEmptyList[A]) extends AnyVal {
```

Comment:
Is there a more widely accessible place this helper could go to?

Reply:
sure